packages/components/src/spinner/style.scss

```diff
@@ -0,0 +1,33 @@
+@keyframes rotation {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+.components-spinner {
+	display: inline-block;
+	background-color: #808080;
```

Comment:
Should we use one of our gray variables?

Reply:
I had the same doubt, but didn't know which one use as none of them matches the color used by the current spinner image.

Any suggestion?

Reply:
This one looks close $dark-gray-200 ?

Reply:
Ah yes that's a good point. Definitely use the SASS colors if you can, we already have an existing ticket to bring those to core: https://core.trac.wordpress.org/ticket/40633

Reply:
It looks nice with that color:

![aug-10-2018 13-44-35](https://user-images.githubusercontent.com/1233880/43956202-8aca8668-9ca3-11e8-8696-c4e59b57f402.gif)
(Left: Original Image - Right: New CSS only)